pkg/bwagreement/config.go

```diff
@@ -20,13 +20,14 @@ var (
 // Config is a configuration struct that is everything you need to start an
 // agreement receiver responsibility
 type Config struct {
-	DatabaseURL string `help:"the database connection string to use" default:"$CONFDIR/agreements.db"`
+	DatabaseURL    string `help:"the database connection string to use" default:"$CONFDIR/agreements.db"`
```

Comment:
Use single config with `"postgres://host:port/params"`. It's easier to configure that way.